src/tests/replace/replace-benchmark.cu

```diff
@@ -0,0 +1,110 @@
+/*
```

Comment:
This should be moved to a separate `bench` folder as we discussed in the binary ops PR review?

Reply:
And separated into benchmarks and unit tests?

Reply:
done